intelsdi-x/snap-plugin-collector-processes
# Add PID namespace entry

Add command line metric


Fixes #29 and #35 

Summary of changes:
- Changed namespace structure to `<process_name>/<process_pid>/<metric>`
- Process name is gathered from `cmdline` (no 15 character limit) or `status` line (15 character limit) for kernel processes (for kernel processes, cmdline is empty)
- Added `<process_name>/ps_count` metric for number of process instances
- Added `<process_name>/all/<metric>` for aggregated metrics
- Updated plugin for `snap-plugin-lib-go` support

How to verify it:
- Launch example task and check collected data

Testing done:
- Unit tests
- Manual testing
